Modules/Devices/src/Devices.Domain/Entities/Identities/IdentityDeletionProcess.cs

```diff
@@ -53,6 +53,13 @@ private void Approve(DeviceId createdByDevice)
         GracePeriodEndsAt = SystemTime.UtcNow.AddDays(IdentityDeletionConfiguration.LengthOfGracePeriod);
     }
 
+    private void Cancel(DeviceId cancelledByDevice)
```

Comment:
Private methods should always be placed below the public methods they belong to.

Also, methods in general should never be placed above the properties.

Reply:
@NikolaVetnic already fixed up this file. In merge conflict I will adjust my changes in right places.